products/BellHybrid/apps/common/include/common/widgets/TimeSetSpinnerVertical.hpp

```diff
@@ -70,8 +69,12 @@ namespace gui
 
         void handleContentChanged() override;
 
-        UIntegerSpinnerFixed *hour   = nullptr;
-        UIntegerSpinnerFixed *minute = nullptr;
+        gui::HBox *hBoxHours              = nullptr;
+        gui::HBox *hBoxMinutes              = nullptr;
+        gui::TextFixedSize *hour_first   = nullptr;
+        gui::TextFixedSize *hour_second   = nullptr;
+        gui::TextFixedSize *minute_first   = nullptr;
+        gui::TextFixedSize *minute_second   = nullptr;
```

Comment:
I think we should stick to snakeCase variable names

Reply:
Changed